report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,38 +1,60 @@
+    \label{matrix:masked_intensity}
 \end{itemize}
 
-The model outputs are the quantified chemical compositions of geological samples. These are primarily the concentrations of major elements, represented in weight percentage. While background elements are also present, our current analysis does not quantify these.
-Our goal is to construct a mapping function $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^O$, where $N$ represents the dimensionality of the processed \gls{libs} signals and $O$ represents the number of target oxides. This function maps a processed \gls{libs} signal vector $\mathbf{x} \in \mathbb{R}^N$ to a vector $\mathbf{v} \in \mathbb{R}^O$ of estimated oxide concentrations. The vector $\mathbf{x}$ is derived from matrix $E$, representing processed spectral data:
+The model outputs are the quantified chemical compositions of geological samples. 
+These are primarily the concentrations of major elements, represented in weight percentage. 
+While background elements are also present, our current analysis does not quantify these. 
+Our goal is to construct a mapping function $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^{n_o}$, where $N$ represents the dimensionality of the processed \gls{libs} signals and $n_o$ denotes the number of target oxides in the Concentration Matrix $\mathbf{C}[\chi, o]$.
+This function maps a processed \gls{libs} signal vector $\mathbf{x} \in \mathbb{R}^N$ to a vector $\mathbf{v} \in \mathbb{R}^{n_o}$ of estimated oxide concentrations. The vector $\mathbf{x}$ is derived from matrix $\mathbf{M}$, representing processed spectral data:
 \[
 \mathbf{v} = \mathcal{F}(\mathbf{x}).
 \]
 
+As previously introduced, quantifying chemical compositions from \gls{libs} spectral data poses significant challenges, including high dimensionality and multicollinearity of the data, as well as matrix effects. Here, we further examine these challenges:
+
+\begin{itemize}
+    \item \textbf{Data Dimensionality and Collinearity:} The high dimensionality of spectral data, coupled with multicollinearity—where multiple spectral features exhibit strong correlations—complicates the modeling and analysis. 
+    This challenge arises both in the Intensity Matrix $\mathbf{I[\chi, l, s, w]}$ and consequently the Averaged Intensity Matrix $\mathbf{A[\chi, l, w]}$, where the high number of wavelengths leads to a large number of dimensions, and the intensity readings across these wavelengths often display high correlations.
+    For instance, emission lines from the same element can overlap, causing their intensity readings at different wavelengths to be interdependent\cite{andersonImprovedAccuracyQuantitative2017}. 
+    This interdependency complicates the extraction of independent spectral features necessary for accurate quantitative analysis.
```

Comment:
> This interdependency complicates the extraction of independent spectral features necessary for accurate quantitative analysis.

What do you mean? Can you explain this to me? Not sure I understand.

Reply:
```suggestion
    This interdependency makes it difficult to isolate and identify unique spectral features, which is crucial because accurate quantitative analysis relies on distinguishing individual contributions of each feature to correctly determine the composition of the sample
```

What do you think about this instead?